include/nccl_ofi_rdma.h

```diff
@@ -345,12 +345,18 @@ typedef struct nccl_net_ofi_rdma_send_comm {
 	/* Comm ID provided by remote endpoint */
 	uint64_t remote_comm_id;
 
+	/* Request tracking connect message */
+	nccl_net_ofi_rdma_req_t *send_conn_req;
+
+	/* Indicates if connect message was delivered (and req freed) */
```

Comment:
Put the two state flags next to each other and put the two requests next to each other.  Use order of fields to indicate how to think about relationships.